src/settings.rs

```diff
@@ -46,6 +46,7 @@ pub struct Settings {
     #[serde(skip)]
     pub debug_outlines: bool,
     pub fps_counter: bool,
+    pub locale: Language,
```

Comment:
Should also increment the save file version and assign a default to incoming files that don't have one set.